examples/rng/run_time_dispatching/CMakeLists.txt

```diff
@@ -29,12 +29,16 @@ set(DEVICE_FILTERS "")
 if(ENABLE_MKLCPU_BACKEND)
   list(APPEND DEVICE_FILTERS "cpu")
 endif()
-if(ENABLE_MKLGPU_BACKEND)
+# RNG only supports CPU backend on Windows
```

Comment:
CPU => MKLCPU

Reply:
updated in [effbf47](https://github.com/oneapi-src/oneMKL/pull/200/commits/effbf4756f37a98d7280b924cdf23da85c1f9a51)